src/cuenta_usuario/views.py

```diff
@@ -24,7 +24,7 @@ class RegistroUsuarioView(MensajeResultadoFormMixin, CreateView):
 
 
 class ActualizarUsuarioView(VistaRestringidaMixin, MensajeResultadoFormMixin, UpdateView):
-    usuarios_permitidos = [TipoUsuario.CLIENTE]
+    usuarios_permitidos = [TipoUsuario.CLIENTE, TipoUsuario.ADMINISTRADOR]
```

Comment:
cambia el permiso para aceptar todos
```suggestion
    usuarios_permitidos = VistaRestringidaMixin.todos_los_usuarios
```